docs/getting_started.md

```diff
@@ -37,6 +37,12 @@ export SCALE_API_KEY="[Your API key]"
 
 You can also add in the line above to your `.zshrc` or `.bash_profile` so it's automatically set for future sessions.
 
+Alternatively, you can also set your API key using either of the following patterns:
```

Comment:
maybe being explicit that this is helpful in nodebook environment?

Reply:
will add that in now